packages/e2e-tests/authenticated/passport-02.test.ts

```diff
@@ -0,0 +1,62 @@
+import test, { expect } from "@playwright/test";
+
+import { openDevToolsTab, startTest } from "../helpers";
+import { E2E_USER_1_API_KEY } from "../helpers/authentication";
+import { warpToMessage } from "../helpers/console-panel";
+import {
+  activateInspectorTool,
+  inspectCanvasCoordinates,
+  openElementsPanel,
+  waitForElementsToLoad,
+} from "../helpers/elements-panel";
+import { openNetworkPanel } from "../helpers/network-panel";
+import { findNetworkRequestRow } from "../helpers/network-panel";
+import { isPassportItemCompleted } from "../helpers/passport";
+import { getReactComponents, openReactDevtoolsPanel } from "../helpers/react-devtools-panel";
+import { enablePassport } from "../helpers/settings";
+import { resetTestUser, waitFor } from "../helpers/utils";
+
+const url = "cra/dist/index.html";
+
+test(`authenticated/passport-02: ...`, async ({ page }) => {
+  await resetTestUser("frontende2e1@replay.io");
+
+  await startTest(page, url, E2E_USER_1_API_KEY);
+
+  await enablePassport(page);
+
+  expect(await isPassportItemCompleted(page, "Inspect UI elements")).toBeFalsy();
+  expect(await isPassportItemCompleted(page, "Inspect network requests")).toBeFalsy();
+  expect(await isPassportItemCompleted(page, "Inspect React components")).toBeFalsy();
+  expect(await isPassportItemCompleted(page, "Jump to code")).toBeFalsy();
+
+  await openDevToolsTab(page);
+  await warpToMessage(page, "Added an entry");
+
+  await openElementsPanel(page);
+  await waitForElementsToLoad(page);
+  await activateInspectorTool(page);
+  await inspectCanvasCoordinates(page, 0.05, 0.01);
+
+  await waitFor(async () =>
+    expect(await isPassportItemCompleted(page, "Inspect UI elements")).toBeTruthy()
+  );
+
+  await openNetworkPanel(page);
+  const networkRequest = await findNetworkRequestRow(page, { name: "index.html" });
+  await networkRequest.click();
+
+  await waitFor(async () =>
+    expect(await isPassportItemCompleted(page, "Inspect network requests")).toBeTruthy()
+  );
+
+  await openReactDevtoolsPanel(page);
+  await getReactComponents(page).nth(0).click();
+
+  await waitFor(async () =>
+    expect(await isPassportItemCompleted(page, "Inspect React components")).toBeTruthy()
+  );
+
+  // TODO add test for the "Jump to code" item.
+  // Currently we can't make example recordings containing mouse or keyboard events...
```

Comment:
:question: What's the problem here?

Reply:
I _think_ there's a chance they would?

The left-hand sidebar comes from `Session.findMouse/KeyboardEvents`.  I know those don't show up for Cypress recordings, but I _think_ they do for Playwright recordings, because we're telling a real browser to click?